src/components/Clipboard/Clipboard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import {TableCellProps} from "react-virtualized";
+import CopyToClipboard from "react-copy-to-clipboard";
+
+export interface IClipboardProps {
+    copyText: string;
+}
+
+export const ClipboardTable = ({cellData}: TableCellProps): JSX.Element => {
+    return (
+        <>
+            <CopyToClipboard text={cellData}>
+                <span>{cellData}</span>
```

Comment:
Why you don't pass this as children? Then you can have only 1 Component that can accept span, p or whatever required.

Reply:
Not sure if this is what you mean:
```
<CopyToClipboard text={cellData}>
    <>
        <span>{cellData}</span>
        <button className="copy_icon"></button>
    </>
</CopyToClipboard>
```
Passing children like this did not work, and I need "ReactVirtualized__Table__rowColumn" to be the parent div to display copy icon properly.

Changed Clipboard to look nicer

```
<CopyToClipboard text={copyText}>
    <div className="clipboard_container">
         <p>{copyText}</p>
         <button className="copy_icon"></button>
    </div>
</CopyToClipboard>
```

Reply:
Ok then :/